scripts/core/menu/notifications/views/notifications.html

```diff
@@ -41,7 +41,7 @@
                         <div class="content" ng-if="notification.name !== 'notify' && notification.name !== 'user:mention' && notification.name !== 'translated:changed' "
                             ng-click="onNotificationClick(notification)">
                             <time sd-datetime data-date="notification._created"></time>
-                            <p class="text"><b>{{:: notification.user_name || "System" }}</b>: <span sd-activity-message data-activity="notification"></span></p>
+                            <p class="text" ng-class="{'with-link': notification.data.is_link}"><b>{{:: notification.user_name || "System" }}</b>: <span sd-activity-message data-activity="notification"></span></p>
```

Comment:
`<p>` shouldn't be clickable. Use `<a>` or `<button>` depending if it's a link or an action.

Reply:
@tomaskikutis , done, please take a look.